app/helpers/http_requests.js

```diff
@@ -0,0 +1,3 @@
+const axios = require('axios');
+
+exports.getRequest = url => axios.get(url);
```

Comment:
I think doing it like this may be limiting, for this exercise it is fine, but just to have in mind...